commands/info.js

```diff
@@ -56,7 +61,7 @@ function * run (context, heroku) {
         path: `/client/v11/databases/${addon.name}`
       }).catch(err => {
         if (err.statusCode !== 404) throw err
-        cli.warn(`${cli.color.addon(addon.name)} is not yet provisioned.\nRun ${cli.color.cmd('heroku pg:wait')} to wait until the db is provisioned.`)
+        cli.warn(`${cli.color.addon(addon.name)} is not yet provisioned.\nRun ${cli.color.cmd('heroku addons:wait')} to wait until the db is provisioned.`)
```

Comment:
Good question. I realized that `pg:wait` doesn't work well under certain circumstances, so especially when people want to know the status of provisioning, `addons:wait` is the safest option at the moment. However, `pg:wait` doesn't do exactly the same thing as `addons:wait`. For example, `pg:wait` tells you more detailed progress when you created an Postgres addon as fork or follow (https://github.com/heroku/shogun/blob/master/lib/shogun/presenters/resource_client_api_presenter.rb#L125-L149). I'm also thinking to tweak [this provisioning message](https://github.com/heroku/shogun/blob/master/lib/web/mediators/formations/postgres_provisioner.rb#L372) and use `heroku addons:wait`, but I'm not sure if we should "deprecate" `pg:wait` for that reason.